Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -1907,6 +1905,76 @@ public async Task RemoveForkedChainWhenFillBlocksAsyncFail()
             }
         }
 
+        [Fact]
+        public async Task HandleReorgInSynchronizing()
+        {
+            var policy = new BlockPolicy<Sleep>(new MinerReward(1));
+            var miner1 = new Swarm<Sleep>(
+                new BlockChain<Sleep>(
+                    policy,
+                    _fx1.Store
+                ),
+                new PrivateKey(),
+                appProtocolVersion: 1,
+                host: IPAddress.Loopback.ToString()
+            );
+            var miner2 = new Swarm<Sleep>(
+                new BlockChain<Sleep>(
+                    policy,
+                    _fx2.Store
+                ),
+                new PrivateKey(),
+                appProtocolVersion: 1,
+                host: IPAddress.Loopback.ToString()
+            );
+            var receiver = new Swarm<Sleep>(
+                new BlockChain<Sleep>(
+                    policy,
+                    _fx3.Store
+                ),
+                new PrivateKey(),
+                appProtocolVersion: 1,
+                host: IPAddress.Loopback.ToString()
+            );
+
+            foreach (var i in Enumerable.Range(0, 8))
+            {
+                miner1.BlockChain.StageTransactions(
+                    new[]
+                    {
+                        Transaction<Sleep>.Create(
+                            0,
+                            new PrivateKey(),
+                            actions: new[] { new Sleep() }
+                        ),
+                    }.ToImmutableHashSet()
+                );
+                var b = await miner1.BlockChain.MineBlock(miner1.Address);
+                miner2.BlockChain.Append(b);
+            }
+
+            try
+            {
+                await StartAsync(miner1);
+                await StartAsync(miner2);
+
+                await BootstrapAsync(miner2, miner1.AsPeer);
+                await BootstrapAsync(receiver, miner1.AsPeer);
+
+                var t = receiver.PreloadAsync(render: true);
+                await miner1.BlockChain.MineBlock(miner1.Address);
+                await miner2.BlockChain.MineBlock(miner2.Address);
+                Block<Sleep> latest = await miner2.BlockChain.MineBlock(miner2.Address);
+                miner2.BroadcastBlocks(new[] { latest });
+                await t;
```

Comment:
Don't we need to compare `Tip`s of these chains?

```csharp
Assert.Equal(miner1.BlockChain.Tip, miner2.BlockChain.Tip);
Assert.Equal(miner1.BlockChain.Tip, receiver.BlockChain.Tip);
```

Reply:
Added.